content/blog/2021/pyflames_v2.md

```diff
@@ -0,0 +1,35 @@
+Title: PyFlames 2021 v2 - The road to PyCon India 2021
+Date: 2021-08-04 17:00
+Category: 2021
+Slug: pyflames-v2
+Summary: PyCon communities acroos India are coming together again to organize a series of mega meetups as a precursor to PyCon India 2021
+
+[PyCon India 2021](https://in.pycon.org/2021/) is about a month away. [Get your tickets](https://in.pycon.org/2021/#tickets) while they last. As a precursor, Python communities across India are coming together to organize a series of mega meetups called PyFlames 2021.
+
+It's a show of unity from all local Python meetup groups and it will be a series of unique events wherein people from different parts of India would come together to join a common meetup and learn more about their favorite aspects of Pyt
```

Comment:
s/local Python meetup groups/local Python meetup groups and [PyCon India 2021 Outreach partners](https://in.pycon.org/2021/)

Reply:
S/Pyt/Python Programming Language.